pandas/io/sas/sas7bdat.py

```diff
@@ -44,8 +44,8 @@ class SAS7BDATReader(BaseIterator):
     index : column identifier, defaults to None
         Column to use as index.
     convert_dates : boolean, defaults to True
-        Attempt to convert dates to Pandas datetime values.  Note all
-        SAS date formats are supported.
+        Attempt to convert dates to Pandas datetime values.  Note that
+        some rarely used SAS date formats may be unsupported.
```

Comment:
well... it's not easy:) I just took format list [here](http://support.sas.com/documentation/cdl/en/etsug/60372/HTML/default/viewer.htm#etsug_intervals_sect010.htm). The list is called "Frequently Used SAS Date Formats" but i don't know if there're others.